HW1/P2/P2a.py

```diff
@@ -1,3 +1,16 @@
 from P2 import *
 
-# Your code here
+import findspark
+findspark.init()
+
+import pyspark
+sc = pyspark.SparkContext()
+
+tuples = []
+for i in xrange(2000):
+    for j in xrange(2000):
+        tuples.append((i,j))
```

Comment:
I think that instead of doing this for loop, they wanted you to use RDDs and the .cartesian() function for creating these tuples. Here is what I did instead:

`row = sc.parallelize(range(0,2000),10)
column = sc.parallelize(range(0,2000),10)

# this creates all possible combinations of rows and columns
tuples = row.cartesian(column)`